doc/espnet2_tutorial.md

```diff
@@ -308,3 +308,8 @@ To use SSLRs in your task, you need to make several modifications.
       output_size: 80
    ```
 3. Because the shift sizes of different `upstream` models are different, e.g. `HuBERT` and `Wav2Vec2.0` have `20ms` frameshift. Sometimes, the downsampling rate (`input_layer`) in the `encoder` configuration need to be changed. For example, using `input_layer: conv2d2` will results in a total frameshift of `40ms`, which is enough for some tasks.
+4. To achieve online decoding, please employ blockwise Transformer/Conformer encoder in the configuration file. Taking `blockwise Transformer` as an example:
```

Comment:
This is not the correct place to describe online decoding.
This is under self-supervised learning.

Can you more carefully make a document?
My suggestion is to have a new section similar to "Usage of Self-Supervised Learning Representations as feature".
Then, please describe what we have to do during training, and what we should do during inference with some code examples and configuration examples. 
Also, several people are asking questions already. So, please add FAQ section and mention them.

Reply:
Sorry for my mistake.